lib/telemetria.ex

```diff
@@ -326,6 +348,7 @@ defmodule Telemetria do
               },
               %{
                 env: unquote(caller),
+                locals: Keyword.take(binding(), unquote(locals)),
```

Comment:
Technically, this is the only significant change. Everything else are tests, deps updates, etc.